docs/specs/xcmp.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: XCMP Interface
+parent: Specification
+nav_order: 2
+---
+
+# XCMP Interface <!-- omit in toc -->
+
+_NOTE: work in progress_
+
+- [Introduction](#introduction)
+- [Scenarios](#scenarios)
+  - [Alice transfers 21 PolkaETH to Bob on another chain](#alice-transfers-21-polkaeth-to-bob-on-another-chain)
+  - [Alice attempts to exchange 42 DOT for 21 PolkaETH](#alice-attempts-to-exchange-42-dot-for-21-polkaeth)
+- [Asset Identification](#asset-identification)
+- [Outstanding Issues](#outstanding-issues)
+  - [Numeric precision](#numeric-precision)
+
+## Introduction
+
+Our parachain will support cross-parachain token transfers using [XCMP](https://github.com/paritytech/xcm-format/blob/master/README.md).
+
+Since our parachain endeavours to be trustless, only reserve-backed transfers will be supported. Participating chains will need to agree on a parent chain to hold sovereign (reserve) accounts.
+
+## Scenarios
+
+### Alice transfers 21 PolkaETH to Bob on another chain
+
+This transfer is modelled on [Transfer via reserve](https://github.com/paritytech/xcm-format/blob/master/README.md#transfer-via-reserve).
+
+Parties:
+- H: Home chain (Polkadot-Ethereum Bridge)
+- D: Destination chain
+- R: Reserve chain
+
+Effects:
+1. H will withdraw 21 PolkaETH from Alice's local account.
+2. The sovereign account of H on R will be reduced by 21 PolkaETH.
+3. The sovereign account of D on R will be credited with 21 PolkaETH.
+4. D will mint 21 PolkaETH into Bob's account.
+
+### Alice attempts to exchange 42 DOT for 21 PolkaETH
```

Comment:
this example makes sense but i dont think our chain will support exchanging assets, so may not be relevant to include

Reply:
yeah, good point. I was just looking for more possible use cases beyond transferring PolkaETH/ERC20 to other parachains (and vice-versa).

I remove the use case